Sources/PrimerSDK/Classes/User Interface/Root/RootViewController+Router.swift

```diff
@@ -10,6 +10,7 @@
 import UIKit
 
 protocol RouterDelegate: class {
+    var root: RootViewController? { get }
```

Comment:
following my comment above, should we call it root or rvc or rootViewController?

Reply:
Leave it root, that's fine. On the later version let's call it rvc.